### PR TITLE
Add ascii @ Dresden, Germany

### PR DESCRIPTION
**Pull request by kiliankoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -326,6 +327,19 @@ Wifi | Power | Address | Open Hours
...
+#### ascii
+
+> Café inside the faculty of Computer Science of the Dresden University of Technology; 
+> always filled with friendly and helpful people, no matter the topic 
+
+Webpage: [ascii-dresden.de](http://ascii-dresden.de)
+
+Wifi | Power | Address | Open Hours
+---- | ----- | ------- | ----------
+✔ (eduroam) | ✔ | [Nöthnitzerstr. 46, 01187 Dresden](https://goo.gl/maps/74VwX2opdDG2) | Mon-Thu 9:00-17:00; Fri 9:00-15:00
```
Comment (RichardLitt) [inline README.md]: Not everyone can access eduroam; is there a different wifi available there?

**Comment by hoodie** [comment]
shouldn't we then also add the GCHQ? @Mic92

**Comment by Mic92** [comment]
of course this is the old venue: https://wiki.hackerspaces.org/C3D2_HQ

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Maybe just remove the check mark, and leave `eduroam` there.

Comment (diasdavid) [inline README.md]: or add `(eduroam only)`

**Comment by kiliankoe** [comment]
@hoodie https://github.com/diasdavid/awesome-hacking-spots#hackerspaces

File: README.md (hunk 1 above)
Comment (kiliankoe) [inline README.md]: good point, I'll update that

**Comment by RichardLitt** [comment]
Looks good to me, but @diasdavid should confirm.

**Comment by diasdavid** [comment]
rad, thank you :)

**Pull request merged**